scss/bootstrap/overrides/style/_buttons.scss

```diff
@@ -103,6 +103,18 @@
   }
 }
 
+.btn-default-outline {
+  background-color: transparent !important;
```

Comment:
There is a `button-outline-variant` mixin you can see being used just below. I think you should use that